jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -12,32 +12,22 @@ pipeline {
         }
     }
     parameters {
-        string(
-            name: 'BUILD_ID',
-            description: 'Build ID of the distribution build manifest generated.',
-            trim: true
-        )
         string(
             name: 'VERSION',
-            description: 'Version of the distribution build manifest.',
+            description: 'Version of the release to download the artifacts.',
```

Comment:
Nit: `Version of the release` should be good. `To download the artifacts` is confusing to me